packages/fuels/Forc.toml

```diff
@@ -60,8 +60,10 @@ members = [
   'tests/types/contracts/enum_encoding',
   'tests/types/contracts/enum_inside_struct',
   'tests/types/contracts/evm_address',
-  'tests/types/contracts/generics',
+  # TODO: @hal3e tempoarily disabled this forc project
+  #'tests/types/contracts/generics',
```

Comment:
Just double checking, is this the one that takes a long time to compile with the new encoder?

Reply:
I delete it for now. Made an issue to bring it back: https://github.com/FuelLabs/fuels-rs/issues/1316